src/bundle/Resources/public/css/modules/table-data.css

```diff
@@ -32,7 +32,7 @@
     display: inline-block;
 }
 
-.ez-table-data td:last-of-type {
+.ez-table-data-row-tool td:last-of-type {
```

Comment:
> ez-table-data-row-tool is more generic than ez-table-checkboxes.

actually, that's 2 different things :) A table can have checkboxes and/or buttons on each row or none or only one of them. So if there's something special with table have checkboxes, there'll probably be `ez-table-data-row-checkbox` or something like that.